mlflow/transformers/__init__.py

```diff
@@ -766,23 +861,19 @@ def log_model(
                         "model": model,
                         "tokenizer": tokenizer,
                     }
-                    mlflow.transformers.log_model(
+                    mlflow.transformers.save_model(
                         transformers_model=components,
-                        artifact_path="my_model",
+                        path="path/to/save/model",
                     )
 
-            An example of submitting a `Pipeline` from a default pipeline instantiation:
+            An example of specifying a local checkpoint path is shown below:
```

Comment:
Instead of replacing the example, can we add example instances of each of the ways to log a transformers model?

Reply:
nah this is not a replacement, see L835~, just weird github diff show

Reply:
gotcha gotcha :)